de_web_plugin.cpp

```diff
@@ -5944,6 +5946,7 @@ void DeRestPluginPrivate::updateSensorNode(const deCONZ::NodeEvent &event)
                                         i->modelId().startsWith(QLatin1String("ZGRC-KEY")) || // Sunricher wireless CCT remote
                                         i->modelId().startsWith(QLatin1String("ZG2833K")) || // Sunricher remote controller
                                         i->modelId().startsWith(QLatin1String("SV01-"))) // Keen Home vent
+                                        i->modelId().startsWith(QLatin1String("4512703")) || // Namron 4-ch remote controller
```

Comment:
I haven't compiled it yet, but I guess this line and the one in line 5914 should throw a compile errors?

Reply:
Sorry, I just copy pasted the code of the similar switch. I have updated it below, and I think that fixes it.